include/enum_quda_fortran.h

```diff
@@ -45,14 +45,16 @@
 #define QUDA_FLOAT_GAUGE_ORDER 1
 #define QUDA_FLOAT2_GAUGE_ORDER 2 //no reconstruct and double precision
 #define QUDA_FLOAT4_GAUGE_ORDER 4 //8 and 12 reconstruct half and single
-#define QUDA_QDP_GAUGE_ORDER 5 //expect *gauge[4] even-odd spacetime row-column color
-#define QUDA_QDPJIT_GAUGE_ORDER 6 //expect *gauge[4] even-odd spacetime row-column color
-#define QUDA_CPS_WILSON_GAUGE_ORDER 7 //expect *gauge even-odd spacetime column-row color
-#define QUDA_MILC_GAUGE_ORDER 8 //expect *gauge even-odd mu spacetime row-column order
-#define QUDA_MILC_SITE_GAUGE_ORDER 9 // packed into MILC site AoS [even-odd][spacetime] array, and [dir][row][col] inside
-#define QUDA_BQCD_GAUGE_ORDER 10 //expect *gauge mu even-odd spacetime+halos row-column order
-#define QUDA_TIFR_GAUGE_ORDER 11
-#define QUDA_TIFR_PADDED_GAUGE_ORDER 12
+#define QUDA_NATIVE_GAUGE_ORDER 5 // used to denote one of the above types in a trait, not used directly
+#define QUDA_QDP_GAUGE_ORDER 6    // expect *gauge[4] even-odd spacetime row-column color
+#define QUDA_QDPJIT_GAUGE_ORDER 7 // expect *gauge[4] even-odd spacetime row-column color
+#define QUDA_CPS_WILSON_GAUGE_ORDER 8 // expect *gauge even-odd spacetime column-row color
+#define QUDA_MILC_GAUGE_ORDER 9       // expect *gauge even-odd mu spacetime row-column order
+#define QUDA_MILC_SITE_GAUGE_ORDER                                                                                     \
+  10                             // packed into MILC site AoS [even-odd][spacetime] array, and [dir][row][col] inside
```

Comment:
This file must not get touched during a Jenkins test, because this certainly won't compile...

Reply:
Eventually, I would like to have this file auto generated from the c equivalent header to remove potential for errors like this.